blueocean-dashboard/src/main/js/components/PipelinePage.jsx

```diff
@@ -44,17 +52,18 @@ export default class PipelinePage extends Component {
                         <TabLink to="/pr">Pull Requests</TabLink>
                     </PageTabs>
                 </PageHeader>
-                {React.cloneElement(this.props.children, { pipeline })}
+                {this.props.children}
             </Page>
         );
     }
 }
 
 PipelinePage.propTypes = {
-    children: object,
+    children: PropTypes.any,
```

Comment:
why not add above any? IMO the code you refactor here is much cleaner then the one you deliver instead. It is obvious that object/any is PropTypes since it is use exclusively within XXX.propTypes. I drop the redundant PropTypes